.github/workflows/test.yml

```diff
@@ -1,6 +1,28 @@
 on: push
 name: proj ci
 jobs:
+  # The `ci-result` job doesn't actually test anything - it just aggregates the
+  # overall build status for bors, otherwise our bors.toml would need an entry
+  # for each individual job produced by the job-matrix.
+  #
+  # Ref: https://github.com/rust-lang/crater/blob/9ab6f9697c901c4a44025cf0a39b73ad5b37d198/.github/workflows/bors.yml#L125-L149
+  #
+  # ALL THE SUBSEQUENT JOBS NEED THEIR `name` ADDED TO THE `needs` SECTION OF THIS JOB!
+  ci-result:
```

Comment:
> So either bors has some caching bug or I am not smart enough to understand the nuanced difference between what I did on my fork vs here.
> 
> I'm going to merge in hopes that a fresh pr will incorporate the new bors.toml (and verify).

Ok - seemed to work! https://github.com/georust/proj/pull/45#event-3826355193

You still might want to take a look at this new config. I

t's non-obvious, but seems less bad than reverse engineering the naming scheme output by the build matrix for our 10 builds.

<img width="315" alt="Screen Shot 2020-09-30 at 1 47 19 PM" src="https://user-images.githubusercontent.com/217057/94737915-78c75480-0323-11eb-9807-5e5402b3bf26.png">